README.md

```diff
@@ -2,30 +2,29 @@
 
 
 ## Overview
+OpenFin Notifications provides developers with a uniform way to create, display and organize desktop notifications as well as responding to notification events.
 
-OpenFin Notifications provide developers with a uniform way to create, display and organize desktop notifications as well as responding to notification events.
-
-Notifications will be displayed as toasts as well as being listed and organized in a Notification Center. The Notification Center can be accessed by clicking on the system tray notification icon.
+Notifications will be displayed as toasts as well as being listed and organized in a Notification Center. The Notification Center can be accessed by clicking on the Notifications icon in the system tray.
 
 This project consist of 3 parts:
-1. The Notifications Service, displaying and managing notifications and owning the Notification Center UI
-2. The Notifications Client, exposing API's for applications to create and manage notifications
+1. The Notifications Provider, displaying and managing notifications, and owning the Notification Center
+2. The Notifications Client, exposing APIs for applications to create and manage notifications
 3. The Notifications Demo App, demonstrating the different features of OpenFin Notifications
 
 ### Dependencies
 - OpenFin version for applications using Notifications >= 9.61.38.41
-- OpenFin version used in the Notifications Service = 10.66.39.43
+- OpenFin version used in the Notifications service = 10.66.39.43
 - RVM >= 4.7
 
 ### Features
 * Create notifications
-* Clear/dismiss notifications
-* Attach handlers for create/close events and notification interactions
+* Clear and dismiss notifications
+* Attach handlers for create and close events, and notification interactions
 * Persist notifications in the Notification Center
 
 ## Getting Started
 
-Integrating the Notifications Service within an application is done in two steps. Add the service to the application manifest, and import the API:
+Integrating the Notifications service within an application is done in two steps - adding the service to the application manifest, and importing the API:
```

Comment:
While not strictly incorrect, I think a colon `:` would be a better fit here than an em-dash `-`. 

Also the comma is unnecessary and the sentence should probably end with a full-stop (period for the Americans).

Reply:
Revised this a bit, feels less awkward now, I think